src/osrm/__main__.py

```diff
@@ -0,0 +1,37 @@
+import site
+import subprocess
+import sys
+
+if(len(sys.argv) < 2):
+    print("Argument not provided")
+    exit()
+
+execpath = site.getusersitepackages()+"/bin/"
+
+if(sys.argv[1] == "components"):
+    execpath += "osrm-components"
+
+elif(sys.argv[1] == "contract"):
+    execpath += "osrm-contract"
+
+elif(sys.argv[1] == "customize"):
+    execpath += "osrm-customize"
+
+elif(sys.argv[1] == "datastore"):
+    execpath += "osrm-datastore"
+
+elif(sys.argv[1] == "extract"):
+    execpath += "osrm-extract"
+
+elif(sys.argv[1] == "partition"):
+    execpath += "osrm-partition"
+
+elif(sys.argv[1] == "routed"):
+    execpath += "osrm-routed"
+
+args = [execpath]
+
+for i in range(2, len(sys.argv)):
+    args.append(sys.argv[i])
+
+subprocess.call(args)
```

Comment:
It'd be better to use smth that can capture stdout & stderr, e.g.

```suggestion
# will stream any output to the shell
# shell=True is safe here and lets people use "~" in paths etc
proc = subprocess.run(args, encoding="utf-8", capture_output=True, shell=True)
sys.exit(proc.returncode)
```

Reply:
After testing this, I wasn't able to get `subprocess.run` to work properly. `capture_output=True` results in no output, and disabling that causes it the executables to not work for me. Did we miss something here?

Reply:
if capture_output=True, stdout/err are only available in `proc.stdout/err` it seems

Reply:
> If I try other commands it works. But I think you're right, capture_output should be False, only then it streams to stdout/err (misunderstood the docs there) and one gotcha is, with `shell=True` you'll need to pass one string, not a list of strings, as command. You can directly edit the sources in your installation tree, since this is only python, and try if that makes it work.

Oh, in that case, I can add a change to it to work as a single string rather than an array.